packages/eslint-config-kibana/typescript.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
       ],
 
       settings: {
+        'import/resolver': {
+          node: {
+            extensions: ['.mjs', '.js', '.json', '.ts', '.tsx'],
```

Comment:
@elastic/kibana-operations extended default settings for `import/resolver` to support typescript files. 
https://github.com/benmosher/eslint-plugin-import/blob/master/resolvers/node/index.js#L31